articles-4.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>

```

Comment:
Another duplicate of Page 1 with adjusted nav buttons at the bottom, and I ensured that the "next" button was hidden as this is the last article list page.